### ISSUE TITLE
Invalid items in the v2.3.0 changelog

### ISSUE THREAD

**Issue by branchmispredictor** [traefik/traefik#7338]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Updated to Traefik v2.3.0 and try to use the newly added ServersTransport config as mentioned in the changelog as: `[service,tls] add ServersTransport on services (#7203 by juliens)`.

Added the following to a service's labels in docker-compose:
```
            - "traefik.http.serversTransports.server.serverName=server"
            - "traefik.http.serversTransports.server.rootCAs=/certs/rootCA.pem"
```

Additionally, looked for the commit in question on the v2.3.0 branch.

### What did you expect to see?

I expected to see Traefik starting without issue, and for the commit in the referenced PR to be on the v2.3.0 branch.

### What did you see instead?

Error log: `time="2020-09-23T23:43:43Z" level=error msg="field not found, node: serversTransports" container=server-docker-services-20fab893aebaacb410a1cb61048f881e109260af60c00eb7b1c2fc2cbb74dceb providerName=docker`

The v2.3.0 branch also seems to be missing the commits to implement ServersTransports (while they are on master).

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
$ docker run traefik version
Version:      2.3.0
Codename:     picodon
Go version:   go1.15.2
Built:        2020-09-23T10:55:07Z
OS/Arch:      linux/amd64

```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

N/A

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

N/A

**Comment by ldez** [comment]
Hello,

It's just an error in the generated changelog.

This feature is not a part of the v2.3.

I will fix that.

**Comment by traefiker** [comment]
Closed by #7339.
